f2/src/__tests__/FeedbackPage.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { i18n } from '@lingui/core'
+import { render, cleanup } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import { ThemeProvider } from 'emotion-theming'
+import { I18nProvider } from '@lingui/react'
+import { FeedbackPage } from '../FeedbackPage'
+import canada from '../theme/canada'
+import en from '../locales/en.json'
+import { StateProvider, initialState, reducer } from '../utils/state'
+
+i18n.load('en', { en })
+i18n.activate('en')
+
+describe('<FeedbackPage />', () => {
+  afterEach(cleanup)
+
+  it('renders', () => {
+    render(
+      <MemoryRouter initialEntries={['/']}>
+        <ThemeProvider theme={canada}>
+          <StateProvider initialState={initialState} reducer={reducer}>
+            <I18nProvider i18n={i18n}>
+              <FeedbackPage location={{ search: undefined }} />
```

Comment:
what is `location` for?

Reply:
Oups, must be from the file I used as an example! I'll fix that.